test/sysinfo.jl

```diff
@@ -7,6 +7,8 @@ sprint(Base.Sys.cpu_summary)
 @test Base.Sys.uptime() > 0
 Base.Sys.loadavg()
 
+@test !("fakename" ∈ keys(ENV))
```

Comment:
(I just searched for Julia UUID and found https://github.com/JuliaArchive/UUID.jl)